templates/satspay/display.html

```diff
@@ -1,329 +1,210 @@
-                      {% raw %} {{ charge.timeLeft }} {% endraw %}</span
-                    ></span
-                  >
-                </q-item>
-              </q-item-section>
-            </q-linear-progress>
-          </div>
-        </div>
-      </div>
-      <div class="row q-ml-md q-mt-md q-mb-lg">
-        <div class="col">
-          <div class="row">
-            <div class="col-4 q-pr-lg">Charge Id:</div>
-            <div class="col-8 q-pr-lg">
-              <q-btn flat dense outline @click="copyText(charge.id)"
-                ><span v-text="charge.id"></span
-              ></q-btn>
-            </div>
-          </div>
-          <div class="row items-center">
-            <div class="col-4 q-pr-lg">Total to pay:</div>
-            <div class="col-8 q-pr-lg">
+          </q-item-label>
+          <q-item dense>
+            <q-item-section>
+              <q-item-label>Total to pay:</q-item-label>
+              <!-- <q-item-label caption>Total amount to pay</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="blue">
-                <span v-text="charge.amount" class="text-subtitle2"></span> sat
+                <span v-text="charge.amount" class="text-subtitle2"></span>
+                <span>&nbsp;sats</span>
               </q-badge>
-            </div>
-          </div>
-          <div class="row items-center q-mt-sm">
-            <div class="col-4 q-pr-lg">Amount paid:</div>
-            <div class="col-8 q-pr-lg">
+            </q-item-section>
+          </q-item>
+          <q-separator spaced></q-separator>
+          <q-item dense>
+            <q-item-section>
+              <q-item-label>Amount paid:</q-item-label>
+              <!-- <q-item-label caption>Amount already paid</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="orange">
                 <span v-text="charge.balance" class="text-subtitle2"></span>
-                sat</q-badge
-              >
-            </div>
-          </div>
-          <div v-if="pendingFunds" class="row items-center q-mt-sm">
-            <div class="col-4 q-pr-lg">Amount pending:</div>
-            <div class="col-8 q-pr-lg">
+                <span>&nbsp;sats</span>
+              </q-badge>
+            </q-item-section>
+          </q-item>
+          <q-separator spaced v-if="pendingFunds"></q-separator>
+          <q-item v-if="pendingFunds" dense>
+            <q-item-section>
+              <q-item-label>Amount pending:</q-item-label>
+              <!-- <q-item-label caption>Amount pending on mempool</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="gray">
-                <span v-text="pendingFunds" class="text-subtitle2"></span> sat
+                <span class="text-subtitle2">${pendingFunds} sats</span>
```

Comment:
- is the `${pendingFunds}` expression evaluated inside an HTML?

Reply:
It's javascript! It uses `delimiters` so we don't need to use `{% raw %}`

Reply:
Its JS inside HTML. I didn't know this works.